### DISCUSSION TITLE
How to setup/select Short Link Domain with "Recipe 3: Domain CAN NOT be on Route 53"

### DISCUSSION THREAD

**Discussion by VissuP** [mozilla/hubs#5746] [General Questions & Discussion] [answered]
I am trying to deploy the stack on our internal AWS account where Route 53 is restricted. So I choose **Recipe 3: Domain CAN NOT be on Route 53.** 

So when creating the stack, under the **Domain Configuration** section for **Short Link Domain Name**, I cannot find anything in the drop-down list. 

So how to get this **Short Link Domain Name** for the domain that cannot be on Route 53? Please advice.

Thanks

**Comment by pattersonbl2** [comment, marked as answer]
Currently, there isn't a way around not having to buy a short-link domain in route53.  The top root domain can be outside route 53 but the short link has to be route 53.

**Reply by VissuP** [reply]
Ok .... Thank you
